fs/config/config.go

```diff
@@ -214,8 +214,17 @@ var errorConfigFileNotFound = errors.New("config file not found")
 // automatically decrypt it.
 func loadConfigFile() (*goconfig.ConfigFile, error) {
 	b, err := ioutil.ReadFile(ConfigPath)
+	envpw := os.Getenv("RCLONE_CONFIG_PASS")
 	if err != nil {
 		if os.IsNotExist(err) {
+			if len(configKey) == 0 && envpw != "" {
+				err := setConfigPassword(envpw)
+				if err != nil {
+					fmt.Println("Using RCLONE_CONFIG_PASS returned:", err)
```

Comment:
If you fancy tidying up, this should be `fs.Errorf(nil, ` instead of `fmt.Println(`